src/components/date-picker/date-picker--range.html

```diff
@@ -0,0 +1,24 @@
+<!-- Ranged -->
+<div class="bx--form-item">
+  <div data-date-picker data-date-picker-type="range" class="bx--date-picker bx--date-picker--range">
+    <div class="bx--date-picker__container">
```

Comment:
I think it's: `main-element__element` and `main-element--modifier` ? And then in the SCSS you just don't nest the BEM elements.